components/encryption/src/manager/mod.rs

```diff
@@ -147,17 +160,22 @@ impl Dicts {
         file.method = compat(method);
         self.file_dict.files.insert(fname.to_owned(), file);
         self.save_file_dict()?;
+        ENCRYPTION_FILE_NUM_GAUGE
```

Comment:
let's update `ENCRYPTION_FILE_NUM_GAUGE` only in `Dicts::new()` and `save_file_dict()`.

Reply:
Same as above